x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/edit_field/edit_field.tsx

```diff
@@ -144,6 +142,23 @@ export const EditField = React.memo(({ form, field, allFields, exitEdit }: Props
                         </EuiButtonEmpty>
                       </EuiFlexItem>
                     </EuiFlexGroup>
+
+                    {/* Field path */}
+                    <EuiFlexGroup>
+                      <EuiFlexItem style={{ display: 'flex', flexDirection: 'row' }}>
+                        <CodeBlock padding="small">
+                          {field.path.map((path, i) =>
+                            i < field.path.length - 1 ? (
+                              <span key={i}>
+                                {path} <EuiIcon type="sortRight" />{' '}
```

Comment:
This change doesn't feel like it's within the scope of the search feature. Could we remove it and discuss it separately? I have similar concerns as JL.

Reply:
This is the complexity of working in parallel PR that depend on one another. I needed the changes of #54146 to continue my work here so I created 1 commit out of it and cherry-picked here.

This article https://jg.gg/2018/09/29/stacked-diffs-versus-pull-requests/ has been waiting for a year in my Chrome tabs that I get some time to give StackedDiff a try and maybe remove the confusion you had if it's too cool to be true 😊